fury/ui.py

```diff
@@ -3349,26 +3350,26 @@ def __init__(self, labels, checked_labels=None, padding=1, font_size=18,
         """
         Parameters
         ----------
-        labels : list(string)
+        labels : list(str)
             List of labels of each option.
-        padding : float
+        checked_labels: list(str), optional
+            List of labels that are checked on setting up.
+        padding : float, optional
             The distance between two adjacent options
-        font_size : int
+        font_size : int, optional
             Size of the text font.
-        font_family : str
+        font_family : str, optional
             Currently only supports Arial.
-        position : (float, float)
+        position : (float, float), optional
             Absolute coordinates (x, y) of the lower-left corner of
             the button of the first option.
-        checked_labels: list(string), optional
-            List of labels that are checked on setting up.
         """
 
         self.labels = list(reversed(labels))
         self._padding = padding
         self._font_size = font_size
         self.font_family = font_family
-        self.checked_labels = checked_labels or []
+        self.checked_labels = list(checked_labels) or []
```

Comment:
Oops, I spoke too fast. Just realized this `list(checked_labels)` will fail is it is `None` (i.e.  the default).

Reply:
Better to just do `self.checked_labels = list(checked_labels)` and change the default to `[]` as I suggested. Calling `list` should do a soft copy of the list.

Reply:
right! Why I did not see it too....  So maybe I should just replace `None` by `()`. What do you think?

Reply:
That's even better ^.

Reply:
and `self.checked_labels = list(checked_labels)` right?

Reply:
No need for the `... or ...`

Reply:
yes, I will update that now